data/org.freedesktop.impl.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,188 @@
+      </variablelist>
+    -->
+    <method name="ListShortcuts">
+      <arg type="o" name="handle" direction="in"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+    </method>
+
+    <!--
+        Bind:
+        @handle: Object path for the #org.freedesktop.impl.portal.Request object representing this call
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @shortcut_id: The identifier of the shortcut we intend to register
+        @results: Vardict with the results of the call
+
+        Allows to configure the binding the global shortcut named @p shortcut_id within the @session_handle session to a new shortcut.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>description</term>
+            <listitem><para>
+              The user-readable description of the trigger that the portal selected. This is intended for the app to be able to render the shortcut into the UI, allowing the user to learn the shortcut
+            </para></listitem>
+          </varlistentry>
+      </variablelist>
+    -->
+    <method name="Bind">
+      <arg type="o" name="handle" direction="in"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="shortcut_id" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+    </method>
+
+    <!--
+        BindShortcuts:
+        @handle: Object path for the #org.freedesktop.impl.portal.Request object representing this call
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @shortcuts: The identifier of the shortcuts we intend to register, empty for all shortcuts
+        @results: Vardict with the results of the call
+
+        Shows a dialog with all shortcuts to configure and offers them back to the app when done.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcut_ids a{sv}</term>
+            <listitem><para>
+              The different shortcut ids that have been registered mapped to user-readable descriptions for the app to present that describes how the shortcut can be invoked.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="BindShortcuts">
+      <arg type="o" name="handle" direction="in"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="as" name="shortcuts" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
```

Comment:
Do you have any particular thoughts about documenting a `replace` boolean option? My idea is that it would be sent as part of the `options` vardict, and would control whether the array of shortcuts should be merged or replace the currently active shortcuts.